server/sonar-db-dao/src/test/java/org/sonar/db/ce/CeActivityDaoTest.java

```diff
@@ -153,30 +155,30 @@ public void test_selectByQuery() {
 
     // no filters
     CeTaskQuery query = new CeTaskQuery().setStatuses(Collections.<String>emptyList());
-    List<CeActivityDto> dtos = underTest.selectByQuery(db.getSession(), query, 0, 10);
+    List<CeActivityDto> dtos = underTest.selectByQuery(db.getSession(), query, forPage(1).andSize(10));
```

Comment:
just my 2 cents: the first read is quite disturbing because of the static import. Then no problem for consecutive usages!